presto-main/src/main/java/com/facebook/presto/type/RowType.java

```diff
@@ -66,18 +66,17 @@ public RowType(List<Type> fieldTypes, Optional<List<String>> fieldNames)
     @Override
     public String getDisplayName()
     {
```

Comment:
Updated row type rendering to be similar to Hive.

Here is what Hive shows:
```
array<struct<PRODUCT_ID:bigint,TIMESTAMPS:string>>
```

Here is what Presto shows:
```
array<row<product_id:bigint, timestamps:varchar>>
```